nautobot/extras/urls.py

```diff
@@ -537,4 +538,25 @@
         name="webhook_changelog",
         kwargs={"model": Webhook},
     ),
+    # JobHook
```

Comment:
IIRC the url patterns are more or less alphabetized in this file - can you move this up?